README.md

```diff
@@ -48,7 +48,130 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefact
 )
 ```
 
-## Setting up the accessibility test folder
+## Plugin Setup
+
+There are two different approaches to writing accessibility tests, you may choose the approach that
+suites your team best.
+
+## Automatic accessibility testing 
+
+Automatic accessibility testing allows you to write tests via a single spec, it can do this by
+automatically scanning through your project for views.
+
+### Example Accessibility Spec
```

Comment:
I wonder if we could have an outline of the procedure at the start to summarise the process, or turn it into numbered steps and with the heading "How to setup automatic accessibility testing"
1. Copy the template spec below into your project
2. Run your tests with `sbt clean a11y:test`
3. Your test output will describe what you need to add to your spec to enable testing the page templates it has been able to automatically discover in your project

Reply:
yeah, i like this - and we need `a11y:test` to setup node etc. first time

Reply:
do we also want to highlight that by leaving tests pending, teams can progressively cover and fix all the pages in their service over time - ie. it doesn't have to all be done as a big bang.